docs/src/sections/NavbarSection.js

```diff
@@ -33,7 +33,7 @@ export default function NavbarSection() {
         <h4>Additional Import Options</h4>
         <p>
           The Navbar Header, Toggle, Brand, and Collapse components are available as static properties
-          the <code>{"<Navbar/>"}</code> component but you can also import them directly from
+          like the <code>{"<Navbar/>"}</code> component but you can also import them directly from
```

Comment:
sorry the right word here is actually "on", "properties on the Navbar component"